data/countries/de/he/main-taunus/data.yaml

```
@@ -20,6 +20,16 @@ urls:
   - type: WEBSITE
     url: http://www.gruene-bad-soden.de
 ---
+city: Eschborn
+country: DE
+district: Main-Taunus
+level: DE:ORTSVERBAND
+state: Hessen
+type: REGIONAL_CHAPTER
+urls:
+  - type: WEBSITE
+    url: http://www.gruene-eschborn.de
```

```suggestion
    url: http://www.grueneeschborn.de/
```

Aufgrund von Redirects ist das oben wohl die tatsächliche URL